lazer/publisher_sdk/proto/state.proto

```diff
@@ -87,6 +87,23 @@ enum FeedKind {
     FUNDING_RATE = 1;
 }
 
+// Market session type
+enum MarketSession {
+    REGULAR = 0;
+    PRE_MARKET = 1;
+    POST_MARKET = 2;
+    OVER_NIGHT = 3;
+}
+
+message FeedMarketSchedule {
+    // Which session this applies to
+    MarketSession session = 1;
+    // Optional numeric value (can exist or not)
+    optional int32 min_publishers = 2;
+    // Future field — not required now, can be omitted
+    string market_schedule = 3;
```

Comment:
Please add `optional` to every field (regardless of logical presence) to make sure we get better handling of missing values in codegen.

Can you clarify comments? `min_publishers` is optional but what happens if it's not present? Why is `market_schedule` optional? What's the meaning of the `FeedMarketSchedule` value without a `market_schedule` field?